wellcomeml/viz/visualize_clusters.py

```diff
@@ -1,39 +1,42 @@
+from typing import Optional, Union
+
 import numpy as np
 import pandas as pd
+
 from bokeh.io import output_notebook, reset_output
 from bokeh.models import Legend, Dropdown, ColumnDataSource, CustomJS
 from bokeh.plotting import figure, output_file, show
-from wellcomeml.viz.palettes import (Wellcome33,
-                                     WellcomeBackground, WellcomeNoData)
 from bokeh.layouts import column
 from bokeh.events import MenuItemClick
 
+from wellcomeml.viz.palettes import (Wellcome33, WellcomeBackground, WellcomeNoData)
 
-def visualize_clusters(clustering, filter_list, radius: float = 0.05,
+
+def visualize_clusters(clustering, filter_list: Optional[list] = None,
+                       texts: Optional[list] = None,
+                       radius: float = 0.05,
                        alpha: float = 0.5,
                        plot_width: int = 1000, plot_height: int = 530,
                        output_in_notebook: bool = True,
-                       output_file_path: str = 'cluster_viz.html',
-                       palette: list = Wellcome33):
+                       output_file_path: Optional[str] = None,
+                       palette: Union[list, str] = 'Wellcome33'):
     """
     This function creates a plot of the clusters
 
-    Parameters
-    ----------
-        clustering : class
-        filter_list : list
-        radius : float, default: 0.05
-        alpha : float, default: 0.5
-        plot_width : int, default: 600
-        plot_height : int, default: 600
-        output_in_notebook : bool, default: True
-        output_file_path : str, default: 'cluster_viz.html'
-        palette : list, default: Wellcome33
-
-    Returns
-    -------
-        None
-            Prints a bokeh figure
+    Args:
```

Comment:
Changing docstring to the google format which is the one we use for the rest: https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings

Reply:
Okay! I used another style guide I found on stack overflow. Better this one since it's the standard in WellcomeML